section4/exercises/classMethods.js

```diff
@@ -28,13 +28,47 @@ Make sure to run the file with node in your command line.
 // that the proteins have been changed.
 
 class Burrito {
-  constructor(protein, base) {
+  constructor(protein, base, toppings) {
     this.protein = protein;
     this.base = base;
+    this.toppings = toppings;
+  }
+
+  changeProtein(protein) {
+    this.protein = protein;
+  }
+
+  addTopping(string) {
+    this.toppings.push(string);
+  }
+
+  removeTopping(string) {
```

Comment:
Nice job putting in some research into how to use methods to accomplish your goals! Also, great job using parameters to make your functions dynamic.